File: 7KYU/Growth ofa  Population/solution.cs
using System;

class Arge {
    
    public static int NbYear(int p0, double percent, int aug, int p)
    {
        int years = 0;
        while(p0 < p)
        {
            p0 = (int)Math.Floor(p0 + aug + (p0 * percent  / 100));
            years++;
        }
        
        return years;
    }
}

File: 7KYU/Growth ofa  Population/solution.c
int nbYear(int p0, double percent, int aug, int p) {
    int years = 0;
  
    while(p0 < p) {
      p0 = (int)floor(p0 + aug + (p0 * percent / 100.0));
      years++;
    }
      
    return years;
}